docs/tutorials/structure_function_showcase.ipynb

```diff
@@ -10,33 +10,85 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        id_ens = np.append(id_ens, np.full(num_observations, j))
</pre><p>I believe that np.append in a cycle is an anti-pattern</p><pre class="ql-syntax" spellcheck="false">In [2]: %%time
&nbsp; &nbsp;...: a = np.array([])
&nbsp; &nbsp;...: for i in range(1000):
&nbsp; &nbsp;...: &nbsp; &nbsp; a = np.append(a, i)
&nbsp; &nbsp;...:
CPU times: user 3.74 ms, sys: 2.63 ms, total: 6.37 ms
Wall time: 4.24 ms

In [3]: %%time
&nbsp; &nbsp;...: a = []
&nbsp; &nbsp;...: for i in range(1000):
&nbsp; &nbsp;...: &nbsp; &nbsp; a.append(i)
&nbsp; &nbsp;...: a = np.array(a)
&nbsp; &nbsp;...:
&nbsp; &nbsp;...:
CPU times: user 199 µs, sys: 3 µs, total: 202 µs
Wall time: 205 µs

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>